### ISSUE TITLE
Converting text with numbers and words

### ISSUE THREAD

**Issue by dataBaseError** [markburns/numbers_in_words#4]
When converting a string `10 Million` to numbers the output is 1000000 instead of outputting 10000000.
```
'10 Million'.in_numbers
=> 1000000
```

Similarly if no number words are within the string the result is 0 instead of being 100.
```
'100'.in_numbers
=> 0
```
I'm not sure if this is the desired behavior but it would be nice if it could handle these type of scenarios.

**Comment by dimidd** [comment]
see my latest commit
https://github.com/dimidd/numbers_in_words/commit/dafc82d6303e925ff3f2473a569d867a21306ac4

**Comment by dataBaseError** [comment]
@dimidd Thank you for bring the fork to my attention. I will migrate my issue to the fork.

**Comment by dataBaseError** [comment]
I just noticed that you do not have issues open so I'll continue here. I tested the latest version and it still seems to have an issue with a combination of numbers and number words. E.g. 
```
'100 Million'.in_numbers
=> 1000000
```
Which should evaluate to `100000000`.

However https://github.com/dimidd/numbers_in_words/commit/dafc82d6303e925ff3f2473a569d867a21306ac4 does handle the second scenario to correctly convert the number into a numerical value when no word numbers are present.
